core/src/util/deprecations.ts

```diff
@@ -119,7 +119,7 @@ export function getDeprecations(style: (s: string) => string = styles.highlight)
     rsyncBuildStaging: {
       contextDesc: "Build Staging",
       featureDesc: `The ${style("legacy rsync-based file syncing")} for build staging`,
-      hint: `Do not use ${style("`GARDEN_LEGACY_BUILD_STAGE`")} environment variable in 0.14.`,
+      hint: `Do not use ${style(`GARDEN_LEGACY_BUILD_STAGE`)} environment variable in 0.14.`,
```

Comment:
Garden version 0.14 has not been released, ideally users stop using this feature in 0.13 already.
```suggestion
      hint: `Do not use ${style(`GARDEN_LEGACY_BUILD_STAGE`)}.`,
```